src/common/conditional_compilation/docs/develop_cc_for_new_component.md

```diff
@@ -23,49 +22,50 @@ There are several macros to help enable conditional compilation(CC) feature for
 
 ### 1. Macro in SELECTIVE_BUILD_ANALYZER mode
 #### OV_CC_DOMAINS
-It defines domains for conditional compilation, in which contains 3 macros to define detail domains(simple scope, switch/cases and factories). The implement code is:
 
-https://github.com/openvinotoolkit/openvino/blob/713eb9683f67a5eb07374e96b7bbbfcd971ca69e/src/common/conditional_compilation/include/openvino/cc/selective_build.h#L113-L116
+It defines domains for conditional compilation, which contains three macros to define detail domains (simple scope, switch/cases, and factories). 
+
+[Open the code](https://github.com/openvinotoolkit/openvino/blob/713eb9683f67a5eb07374e96b7bbbfcd971ca69e/src/common/conditional_compilation/include/openvino/cc/selective_build.h#L113-L116)
```

Comment:
Can the code be directly displayed here if use this format?

Reply:
For sure we can, but it will be just a code snippet without showing the place of it in the code file. If we want to just show the code snippet, we can try this.